Users/Install.rst

```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala, that's a different software.
```

Comment:
alright so a few more things and this is done:
1. i think "as that's" sounds a bit better :)
2. commit body: "Added" -> "Adds"
"for not to using" -> "for not using"
end with period.

After this, it should be done :)